sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -44,46 +44,41 @@ az_keyvault_keys_client_options const AZ_KEYVAULT_CLIENT_DEFAULT_OPTIONS
             .delay_in_ms = 30,
         } };
 
-AZ_NODISCARD AZ_INLINE az_span
-az_keyvault_get_json_web_key_type_span(az_keyvault_json_web_key_type const key_type) {
-  switch (key_type) {
-    case AZ_KEYVAULT_JSON_WEB_KEY_TYPE_EC: {
-      return AZ_KEYVAULT_WEB_KEY_TYPE_EC_STR;
-    }
-    case AZ_KEYVAULT_JSON_WEB_KEY_TYPE_EC_HSM: {
-      return AZ_KEYVAULT_WEB_KEY_TYPE_EC_HSM_STR;
-    }
-    case AZ_KEYVAULT_JSON_WEB_KEY_TYPE_RSA: {
-      return AZ_KEYVAULT_WEB_KEY_TYPE_RSA_STR;
-    }
-    case AZ_KEYVAULT_JSON_WEB_KEY_TYPE_RSA_HSM: {
-      return AZ_KEYVAULT_WEB_KEY_TYPE_RSA_HSM_STR;
-    }
-    case AZ_KEYVAULT_JSON_WEB_KEY_TYPE_OCT: {
-      return AZ_KEYVAULT_WEB_KEY_TYPE_OCT_STR;
-    }
-    default: {
-      return az_span_create_empty();
-    }
-  }
-}
-
 /**
  * @brief Action that uses json builder to construct http request body used by create key
  *
  * @param kty required value to create a new key
  * @param write
  * @return AZ_NODISCARD build_request_json_body
  */
-AZ_NODISCARD AZ_INLINE az_result
-build_request_json_body(az_span const kty, az_span_action const write) {
+AZ_NODISCARD AZ_INLINE az_result build_request_json_body(
```

Comment:
Something doesn't feel right to me here. 
First, this is an internal function, right?
Second, this function is ONLY called when creating a key, right? If so, the function name should indicate this.

Reply:
name and doxygen comments updated for this internal method.  Let me know if it still looks weird.
It is just an inline function in charge of building json payload for request